scd/README.md

```diff
@@ -1,5 +1,10 @@
 # Flight planning automated testing interface
 
+---
+Note: This interface will be DEPRECATED and REMOVED.  The replacement interface will be the [flight planning interface](../flight_planning).
```

Comment:
Does it make sense to add `deprecated: true` in the endpoints of the OpenAPI yaml?

Reply:
Added